lib/oidcstrategy.js

```diff
@@ -190,6 +193,12 @@ function Strategy(options, verify) {
   this._verify = verify;
   this._passReqToCallback = !!options.passReqToCallback;
 
+  // For each microsoft login page tab, we generate a {state, nonce, policy, timeStamp} tuple,
+  // normally user won't keep opening microsoft login page in new tabs without putting their 
+  // password for more than 10 tabs, so we only keep the most recent 10 tuples in session.
+  // The lifetime of each tuple is 60 minutes or user specified.
+  this._sessionContentHandler = new SessionContentHandler(10, options.nonceLifetime || 3600);
```

Comment:
Please define the 10 as a constant with a comment on what it's for. This looks like a magic number. It's probably worth defining the 3600 as a constant too.